docs/faq.md

```diff
@@ -0,0 +1,89 @@
+# Frequently Asked Questions
+
+## What is Jupytext?
+
+Jupytext is a Python package that can convert Jupyter notebooks to scripts or Markdown documents. It can also convert these text documents back to Jupyter notebooks.
+
+## Why would I want to convert my notebooks to text?
+
+The text representation have much cleaner diffs than the original notebook format. Merging multiple contributions to a notebook in any of these text formats is easier than with the JSON format. Last but not least, acting on a notebook represented as text (spell check, reformat, ...) is sometimes more comfortable than in Jupyter.
+
+## How do I use Jupytext?
```

Comment:
IMO here you should just link to the "using jupytext" section of the documentation. If that documentation is too wordy and complex, then I'd add this short example as a quick "getting started" section at the top of the "using jupytext" section, rather than in the FAQ